package.json

```diff
@@ -36,12 +36,12 @@
 		"publish-to-npm": "npm install && npm test && npm run coverage && npm publish --access=public"
 	},
 	"dependencies": {
-		"@natlibfi/marc-record": "^3.0.0",
+		"@natlibfi/marc-record": "^4.0.0",
 		"@natlibfi/marc-record-validate": "^4.0.0",
-		"@natlibfi/marc-record-validators-melinda": "^5.1.1",
-		"@natlibfi/melinda-record-import-commons": "^1.5.0",
+		"@natlibfi/marc-record-validators-melinda": "^6.0.0",
+		"@natlibfi/melinda-commons": "^4.0.4",
+		"@natlibfi/melinda-record-import-commons": "^3.1.1",
```

Comment:
Update to the newest version (API authentication has changed and the new version of API client handles this).

Reply:
Updated.